gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1038,6 +1025,95 @@ double ofx_get_investment_amount(const struct OfxTransactionData* data)
     }
 }
 
+// gnc_ofx_process_next_file processes the next file in the info->file_iter list.
+static void
+gnc_ofx_process_next_file (GtkDialog *dialog, gpointer user_data)
+{
+    ofx_info* info = (ofx_info*) user_data;
+    if (info->file_iter->next)
+    {
+        // Done with the previous OFX file, process the next one if any.
+        GSList* temp = info->file_iter;
+        info->file_iter = info->file_iter->next;
+        temp = g_slist_delete_link(temp, temp);
+        gnc_file_ofx_import_process_file (info);
+    }
+    else
+    {
+        // No more OFX file to process, final cleanup.
+        g_free (info);
+        if (ofx_created_commodites)
+        {
+            /* FIXME: Present some result window about the newly created
+             * commodities */
```

Comment:
Is this planned as part of the current PR ?

Reply:
This would also be easy if the ofx importer was transformed into an assistant.

Reply:
It was part of the old code, so I didn't remove it....

Reply:
No problem - it just caught my eye.